recipe/meta.yaml

```diff
@@ -26,7 +29,8 @@ requirements:
     - python
 
 test:
-  imports:
```

Comment:
> The `run_test.py` imports the module so I thought this was redundant.

It is mostly for clarity purposes only. Also, some people fetch conda-forge recipes and do not run `run_test.py` due to limited resources.